services/drupal/web/modules/custom/epa_workflow/epa_workflow.install

```diff
@@ -58,3 +58,202 @@ function epa_workflow_uninstall() {
   }
 
 }
+
+/**
+ * Sets all nodes with broken type reference to “Overviews and Fact Sheets”.
+ */
+function epa_workflow_update_9001(&$sandbox) {
+  $results = \Drupal::database()
+    ->query(
+      "SELECT entity_id
+             FROM node__field_type
+             WHERE field_type_target_id NOT IN (
+                SELECT tid from taxonomy_term_data where vid = 'type'
+             );")
+    ->fetchCol('entity_id');
+
+  if (count($results)) {
+    /** @var \Drupal\node\NodeInterface[] $nodes */
+    $nodes = \Drupal::entityTypeManager()
+      ->getStorage('node')
+      ->loadMultiple($results);
+
+    foreach ($nodes as $node) {
+      $node->set('field_type', ['target_id' => 9]);
+      $node->save();
+    }
+  }
+}
+
+/**
+ * Update all nodes without field_review_deadline value to have value 1 year from now and a scheduled transition.
+ */
+function epa_workflow_update_9002(&$sandbox) {
+  // Set the review deadline value to a year from now
+  $now = \Drupal\Core\Datetime\DrupalDateTime::createFromTimestamp(time());
+  $now->add(new DateInterval('P1Y'));
+  $formatted = $now->format('Y-m-d\TH:i:s');
+
+  if (!isset($sandbox['total'])) {
+    // Query all published nodes that do not have a field_review_deadline value set.
+    $result = \Drupal::entityQuery('node')
+      ->accessCheck(FALSE)
+      ->condition('status', 1)
+      ->notExists('field_review_deadline')
+      ->execute();
+
+    $sandbox['total'] = count($result);
+    $sandbox['current'] = 0;
+  }
+
+  $batch_size = 25;
+
+  $nids = \Drupal::entityQuery('node')
+    ->condition('status', 1)
+    ->notExists('field_review_deadline')
+    ->range($sandbox['current'], $batch_size)
+    ->execute();
+
+  if (empty($nids)) {
+    $sandbox['#finished'] = 1;
+    return;
+  }
+
+  /** @var \Drupal\node\NodeInterface[] $nodes */
+  $nodes = \Drupal::entityTypeManager()
+    ->getStorage('node')
+    ->loadMultiple($nids);
+
+  foreach ($nodes as $node) {
+    $node->set('field_review_deadline', $formatted);
+    $node->save();
+    $sandbox['current']++;
+  }
+
+  \Drupal::messenger()->addStatus($sandbox['current'] . ' nodes without review deadline processed.');
+
+  if ($sandbox['current'] >= $sandbox['total']) {
+    $sandbox['#finished'] = 1;
+  } else {
+    $sandbox['#finished'] = ($sandbox['current'] / $sandbox['total']);
+  }
+}
+
+/**
+ * Update all published nodes missing a transition date with a review deadline
+ * less than 3 weeks away or in the past.
+ */
+function epa_workflow_update_9003(&$sandbox) {
+  // Create dates for 1 year in the future and 3 weeks in the future from now.
+  $now = \Drupal\Core\Datetime\DrupalDateTime::createFromTimestamp(time());
+  $one_year = $three_weeks = $now;
```

Comment:
Unless there's handling in the DrupalDateTime class to avoid this, I believe this approach to assigning objects is going to result in $one_year and $three_weeks both referencing the exact same object, which is going to result in, I believe, the $formatted_three_weeks variable getting a date that is exactly one year and 3 weeks in the future.  Forgive me if I've missed something.